src/Sulu/Component/HttpKernel/SuluKernel.php

```diff
@@ -155,8 +169,10 @@ protected function load(LoaderInterface $loader, $confDir, $pattern)
         $excludedConfigFiles = $this->glob($confDir, $pattern . $reversedConfigExtensions);
 
         foreach ($configFiles as $resource) {
-            if (!\in_array($resource, $excludedConfigFiles)) {
+            if (!\in_array($resource, $excludedConfigFiles) && !\in_array($resource, $this->loadedFiles)) {
```

Comment:
for completion, we need this line to prevent loading the `sulu_admin.yaml` more two times in the `admin` context: this is caused by the following lines:
https://github.com/sulu/sulu/blob/5e0db29766f99ae62528123a5ca78200bbc21be0/src/Sulu/Component/HttpKernel/SuluKernel.php#L85-L92